testapi.pm

```diff
@@ -932,6 +932,7 @@ sub script_output {
         type_string($current_test_script);
         type_string("\n", terminate_with => 'EOT');
         wait_serial("$marker-0-");
+        wait_serial('# ', undef, 0, no_regex => 1);
```

Comment:
Please put this just before the `type_string` which requires it. In the future, someone may decide to replace the below `type_string` with `script_run` which contains a `wait_serial('# ' ...`. Also there is no need to synchronise the test script and os-autoinst at this point.

Reply:
done